runtime/integration-tests/src/generic/config.rs

```diff
@@ -123,35 +151,48 @@ pub trait Runtime:
 		+ From<pallet_investments::Call<Self>>
 		+ From<pallet_loans::Call<Self>>
 		+ From<cumulus_pallet_parachain_system::Call<Self>>
-		+ From<orml_oracle::Call<Self>>;
+		+ From<orml_oracle::Call<Self>>
+		+ From<pallet_preimage::Call<Self>>
+		+ From<pallet_collective::Call<Self, CouncilCollective>>
+		+ From<pallet_democracy::Call<Self>>
+		+ From<pallet_liquidity_pools_gateway::Call<Self>>;
 
 	/// Just the RuntimeEvent type, but redefined with extra bounds.
 	/// You can add `TryInto` and `From` bounds in order to convert pallet
 	/// events to RuntimeEvent in tests.
 	type RuntimeEventExt: Parameter
 		+ Member
-		+ From<frame_system::Event<Self>>
 		+ Debug
 		+ IsType<<Self as frame_system::Config>::RuntimeEvent>
 		+ TryInto<frame_system::Event<Self>>
 		+ TryInto<pallet_balances::Event<Self>>
 		+ TryInto<pallet_transaction_payment::Event<Self>>
 		+ TryInto<pallet_loans::Event<Self>>
 		+ TryInto<pallet_pool_system::Event<Self>>
+		+ TryInto<pallet_liquidity_pools_gateway::Event<Self>>
 		+ From<frame_system::Event<Self>>
 		+ From<pallet_balances::Event<Self>>
 		+ From<pallet_investments::Event<Self>>
 		+ From<pallet_transaction_payment::Event<Self>>
 		+ From<pallet_loans::Event<Self>>
 		+ From<pallet_pool_system::Event<Self>>
-		+ From<orml_oracle::Event<Self>>;
+		+ From<orml_oracle::Event<Self>>
+		+ From<pallet_foreign_investments::Event<Self>>
+		+ From<pallet_investments::Event<Self>>
+		+ From<orml_tokens::Event<Self>>
+		+ From<pallet_liquidity_pools_gateway::Event<Self>>
+		+ From<pallet_order_book::Event<Self>>
+		+ From<pallet_preimage::Event<Self>>
+		+ From<pallet_collective::Event<Self, CouncilCollective>>
+		+ From<pallet_democracy::Event<Self>>;
 
 	type RuntimeOriginExt: Into<Result<RawOrigin<Self::AccountId>, <Self as frame_system::Config>::RuntimeOrigin>>
 		+ From<RawOrigin<Self::AccountId>>
 		+ Clone
 		+ OriginTrait<Call = <Self as frame_system::Config>::RuntimeCall>
 		+ From<pallet_ethereum::RawOrigin>
-		+ Into<Result<pallet_ethereum::Origin, <Self as frame_system::Config>::RuntimeOrigin>>;
+		+ Into<Result<pallet_ethereum::Origin, <Self as frame_system::Config>::RuntimeOrigin>>
+		+ From<pallet_liquidity_pools_gateway::GatewayOrigin>;
```

Comment:
Suppose that it took a while to figure out this. 💯

Reply:
This, not that much, luckily the compiler was super helpful here. I had more fun on the events side ^^